Objects/typeobject.c

```
@@ -4578,14 +4607,12 @@ object___reduce_ex___impl(PyObject *self, int protocol)
     if (objreduce == NULL) {
         objreduce = _PyDict_GetItemId(PyBaseObject_Type.tp_dict,
```

What about `_PyDict_GetItemIdWithError()`?

---

This call should be always successful. It is easier to keep `_PyDict_GetItemId()` and ignore an error than add the handling of errors here. Maybe this will be changed in other attempt.